policy-engine/src/graph.rs

```diff
@@ -45,7 +45,7 @@ pub(crate) async fn index(
     V1_GRAPH_INCOMING_REQS.inc();
 
     // Check that the client can accept JSON media type.
-    commons::ensure_content_type(req.headers(), CONTENT_TYPE)?;
+    commons::validate_content_type(req.headers(), CONTENT_TYPE)?;
```

Comment:
Lets rework `CONTENT_TYPES` into a vector of accepted content-type regexps? This would make this a bit more explicit

Reply:
I'd rather feed it a list of types we were comfortable returning, and have `validate_content_type` handle as much of Accept header parsing as we expect to do, and return the accepted type.  At the moment, we only support returning `application/json`, so I'm fine passing in that single type string and not returning anything besides errors.  Can we punt passing in a list of types into a future if/when we support returning multiple types?

Reply:
Okay, sure thing